rides-and-fares/src/main/java/org/apache/flink/training/exercises/ridesandfares/RidesAndFaresExercise.java

```diff
@@ -99,19 +102,43 @@ public static void main(String[] args) throws Exception {
     public static class EnrichmentFunction
             extends RichCoFlatMapFunction<TaxiRide, TaxiFare, RideAndFare> {
 
+        private ValueState<TaxiRide> taxiRideValueState;
```

Comment:
it should be `transient`

Reply:
There was a discussion about transient states. But because this is a static class, there is no need.

Reply:
Please make it `transient`